docs/sprints/completed/SPRINT_12_ARCHITECTURE_POLISH.md

```
@@ -225,6 +337,13 @@
 - [ ] No new Credo or Dialyzer issues introduced
 - [ ] CI builds remain green throughout sprint
 
+### TODO Resolution
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)